### PR TITLE
Remove tracking tools and only use Segment 

### PR DESCRIPTION
**Pull request by Stormtv**
## High Level Overview of Change
Removes Heap, Hubspot, GA, Sentry and only uses segment now

Tracks GitHub link clicks via the client plugin, tracks documentation page views also.

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: src/plugins/analytics.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  // Emit events for all github link clicks
+  function anchorListener(event) {
+    let element = event.target
+    if (
+      element.tagName == 'A' &&
+      element.href.startsWith('https://github.com')
+    ) {
+      analytics.track('Github Navigation', {
+        type: document.title.split(' | ')[0],
```
Comment (docforcedev) [inline src/plugins/analytics.js:16]: Can this return empty?

File: src/plugins/analytics.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment'
+
+export default (function () {
+  if (!ExecutionEnvironment.canUseDOM) {
+    return null
+  }
+
+  // Emit events for all github link clicks
+  function anchorListener(event) {
+    let element = event.target
```
Comment (docforcedev) [inline src/plugins/analytics.js:10]: Is there any strange case where this can be undefined?

File: src/plugins/analytics.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    if (
+      element.tagName == 'A' &&
+      element.href.startsWith('https://github.com')
```
Comment (docforcedev) [inline src/plugins/analytics.js:13]: Will this ever link to a subdomain like gh pages?

File: src/plugins/analytics.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        title: titleElement.textContent,
+      })
+    }
+  }
+
+  // On any click call the anchor listener event
+  document.addEventListener('click', anchorListener)
+
+  // Wait until next tick to record documentation viewed
+  setTimeout(recordDocumentationPage, 100)
```
Comment (docforcedev) [inline src/plugins/analytics.js:46]: Do we want to record this for the new docs landing page which is a marketing page?

**Review by docforcedev — APPROVED** [review]
@Stormtv Lools good, minor negligible questions.

File: src/plugins/analytics.js (hunk 4 above)
Comment (Stormtv) [reply src/plugins/analytics.js:46]: Not sure on that, that would depend on what the data team wants. I'll reach out on that.

File: src/plugins/analytics.js (hunk 3 above)
Comment (Stormtv) [reply src/plugins/analytics.js:13]: No I think all github pages are at github.io and they have subdomains

File: src/plugins/analytics.js (hunk 2 above)
Comment (Stormtv) [reply src/plugins/analytics.js:10]: I don't believe so

File: src/plugins/analytics.js (hunk 1 above)
Comment (Stormtv) [reply src/plugins/analytics.js:16]: No, docusaurus always pipes the title of the doc with the title from the docusaurus config

**Pull request merged**